.travis.yml

```diff
@@ -33,15 +33,14 @@ before_install:
     # export PATH=$PATH:/usr/local/bin
     # jq --version
     echo "============ setting up test data ============="
-    ./testdata/get_token.sh # creates the token.txt
     ./testdata/seeder.sh # reads the token.txt
     echo "============ test data completed  ============="
 
 script:
   - |
     cat token.txt
     export GITLAB_OAUTH_TOKEN=$(./testdata/get_oauth_token.sh)
-    export GITLAB_PRIVATE_TOKEN=$(cat token.txt)
+    export GITLAB_PRIVATE_TOKEN=$(cat ./testdata/token.txt)
```

Comment:
Create a script like `./testdata/get_oauth_token.sh` that returns an output of the private token. I used `$(cat ./testdata/token.txt)` because `./testdata/get_token.sh` generates `a token.txt` it previously for travisci.